### ISSUE TITLE
ENH-000088862  In Attribute Assistant's Nearest Feature method, allow the user to indicate the part of geometry that will be used to determine the nearest feature; for example, ALLPOINTS | NODES | FROMNODE | TONODE | MIDPOINT.

### ISSUE THREAD

**Issue by MikeMillerGIS** [Esri/local-government-desktop-addins#54]
<div><div>
<span><b>Salesforce ID:</b> ENH-000088862</span><br/>
<span><b>Salesforce Submitter:</b> Kory Kramer</span><br/>
<span><b>Salesforce Submit Date:</b> 7/06/2015 10:31 AM</span><br/>
<span><b>Salesforce Bug Type:</b> Product</span><br/>
<span><b>Salesforce Severity:</b> Medium</span><br/>
<span><b>Repro Data:</b> (n/a)</span><br/>
<span><b>Work Around:</b> 1. What is the industry?<br>Customer #440286 is an urban Local Government around 200.000 inhabitants<br><br>2. What is the user&#39;s workflow and use case?<br>Water distribution network collection with ArcMap.<br><br>3. What is the problem that this software functionality would solve?<br>It would help automate attribute transfer from Road network and zoning to water distribution line features.<br><br>4. What are the benefits or changes that would result from the enhancement?<br>It would increase productivity and database quality<br><br>5. How critical is the enhancement to your organization?<br>Medium -high<br><br>6. How do you believe this enhancement may benefit other organizations?<br>It would give more options and flexibility to Attribute assistant<br><br>7. How many users are affected?<br>5 users<br><br> </span><br/>
<span><b>Product:</b> ArcGIS for Local Government</span><br/>
<span><b>Functional Category:</b> ArcGIS for Local Government</span><br/>
<span><b>Client Platform:</b> Windows 7</span><br/>
<span><b>Version Found:</b> 10.2</span><br/>
<span><b>Planned Version Fixed:</b> (n/a)</span><br/>
<span><b>Comment:</b> (n/a)</span><br/>
</div></div>

**Comment by roemhildtg** [comment]
@MikeMillerGIS is this going to be implemented? This would be a great enhancement. 

We maintain storm and sanitary sewer networks that don't participate in an Esri geometric network. Currently it seems that Attribute Assistant will only update from points if the feature classes participate in a geometric network. 

This enhancement would allow updating of structures from the pipe, etc. without the need to create a geometric network.

**Comment by MikeMillerGIS** [comment]
We recently added S(Start) and E(End) for lines as an option to intersecting feature.  Will this support what you are looking to do?
http://solutions.arcgis.com/shared/help/attribute-assistant/tools/attribute-assistant-methods/#intersect-feature

**Comment by roemhildtg** [comment]
Yes, that looks promising. It looks like I had an outdated copy of the documentation Overview.

**Comment by MikeMillerGIS** [comment]
we just released this update last week.

**Comment by roemhildtg** [comment]
Oh wow, that's great! Glad to know that its an active project. This tool seems like its not well known, but could literally benefit any industry, I just discovered it yesterday.

**Comment by LindsayThomas** [comment]
closing issue to match backend support app
